conda/recipes/rmm/recipe.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2019-2025, NVIDIA CORPORATION.
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: rmm
+  version: ${{ version }}
+
+source:
+  - path: ../../..
+
+build:
+  string: cuda${{ cuda_major }}_py${{ py_version }}_${{ date_string }}_${{ head_rev }}
+  script:
+    file: build.sh
+    secrets:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+
+    env:
+      CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER") }}
+      CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER") }}
+      CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER") }}
+      CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR") }}
+      SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET") }}
+      SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT") }}
+      SCCACHE_REGION: ${{ env.get("SCCACHE_REGION") }}
+      SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL") }}
+      SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS") }}
+      SCCACHE_S3_KEY_PREFIX: rmm-${{ env.get("RAPIDS_CONDA_ARCH") }}
+
+requirements:
+  build:
+    - cmake ${{ cmake_version }}
+    - ninja
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ compiler("cuda") }}
+    - cuda-version =${{ cuda_version }}
+    - ${{ stdlib("c") }}
+  host:
+    - cuda-version =${{ cuda_version }}
+    - if: cuda_major == "11"
+      then:
+        - cudatoolkit
+        - cuda-python >=11.8.5,<12.0a0
+      else:
+        - cuda-cudart-dev
+        - cuda-python >=12.6.2,<13.0a0
+    - cython >=3.0.0
+    - rapids-build-backend >=0.3.0,<0.4.0.dev0
+    - librmm =${{ version }}
+    - python
+    - pip
+    - scikit-build-core >=0.10.0
+  run:
+    - if: cuda_major == "11"
+      then:
+        - cudatoolkit
+        - cuda-python >=11.8.5,<12.0a0
+      else:
+        - cuda-cudart
+        - cuda-python >=12.6.2,<13.0a0
+    - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+    - numpy >=1.23,<3.0a0
+    - python
+  ignore_run_exports:
+    from_package:
+      - ${{ compiler("cuda") }}
+      - cuda-python
+      - if: not (cuda_major == "11")
+        then: "cuda-cudart-dev"
+
+tests:
+  - python:
+      imports:
+        - rmm
+      pip_check: false
+
+about:
+  homepage: ${{ load_from_file("python/rmm/pyproject.toml").project.urls.Homepage }}
+  license: ${{ load_from_file("python/rmm/pyproject.toml").project.license.text | replace(" ", "-") }}
+  summary: ${{ load_from_file("python/rmm/pyproject.toml").project.description }}
```

Comment:
Is it possible to assign `${{ load_from_file("python/rmm/pyproject.toml").project }}` to a global variable and extract each value from it?

Reply:
```suggestion
about:
  homepage: ${{ project.urls.Homepage }}
  license: ${{ project.license.text }}
  summary: ${{ project.description }}
```

Reply:
Not currently, no.  https://github.com/prefix-dev/rattler-build/issues/1423

Reply:
https://github.com/rapidsai/rmm/pull/1796#discussion_r1950235393